### DISCUSSION TITLE
osu!lazer Mouse movement feels sluggish.

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#17839] [Q&A] [answered]
So the mouse movement for osu!lazer feels a lot more slow or sluggish than stable, I use basically the same settings from stable, on lazer. Any reason why it feels different?

Settings:
Sensitivity: 1.00x
Raw Input: Enabled
DPI: 2k

Specs: 
Logitech G603
(I play on laptop)

**Comment by Theighlin** [comment, marked as answer]
Have you read #15188 and #16869? As it is this discussion is unactionable because there is not enough information to go by to understand the problem.

**Reply by ghost** [reply]
It’s not like I can accurately compare mouse movement data between stable and lazer, I am not a developer after all and two I don’t know much about logs to say that they would be helpful, but I will go grab some here in a bit.

**Reply by ghost** [reply]
There does not seem to be any relevant logs shown in the log folder.
![Screenshot_4](https://user-images.githubusercontent.com/88257037/163681116-8fbacacd-4206-473c-98a6-925cd58fcee1.png)

**Reply by bdach** [reply]
please upload them and let us decide whether they're relevant or not.

a video of this may also help, with the frame graph showing, if possible (press ctrl-f11 twice).

**Reply by ghost** [reply]
I shall do that then

**Reply by ghost** [reply]
[legacy-ipc.log](https://github.com/ppy/osu/files/8500182/legacy-ipc.log)
[performance.log](https://github.com/ppy/osu/files/8500183/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8500184/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8500185/updater.log)
[database.log](https://github.com/ppy/osu/files/8500186/database.log)

**Reply by ghost** [reply]
Give me like 5 minutes to record the video

**Reply by ghost** [reply]
https://www.youtube.com/watch?v=I29kvOQiV1I

**Reply by bdach** [reply]
i see that you are running lazer single threaded. why is that the case? can you try choosing multi-threading in settings and see if it helps?

**Reply by ghost** [reply]
ok I shall try that though I was already running multi threaded before.

**Reply by ghost** [reply]
The input did indeed go down but it just does not feel the same as stable.

**Reply by bdach** [reply]
we can't comment on or do anything useful with "feel" sorry.

**Reply by ghost** [reply]
https://youtu.be/ll_F6NrbAnY

**Reply by bdach** [reply]
i don't see anything out of the ordinary in the last video, sorry.

**Reply by ghost** [reply]
Well that's unfortunate. Guess Ill stick to stable.

**Reply by ghost** [reply]
Ah but you did see the last few minutes of the first video correct?

**Reply by bdach** [reply]
i did see the part where you recorded stable yes. i'm not sure what i'm supposed to be seeing there.

**Reply by ghost** [reply]
Then forget it.

**Reply by integerrr** [reply]
Judging from the video I'm fairly certain you are using v-sync, try 2x refresh rate

**Reply by ghost** [reply]
It’s a little better with 2x refresh rate. But now it looks like I’m either having minor screen tearing/ minor lag

**Reply by ghost** [reply]
So I tried unlimited and it works a lot better but I feel it lack by a very small margin, I’m sure this will improve once the game is optimized more, or I just get better hardware.

**Reply by integerrr** [reply]
fwiw if your hardware is struggling to keep 240 fps on the update thread even on vsync then I doubt unlimited would do you any good but to each their own

**Reply by ghost** [reply]
Ok that was a lie I still feel the difference. Question is Raw Input on lazer different from stable in anyway?

**Comment by peppy** [comment]
Probably running non-exclusive fullscreen.

**Reply by ghost** [reply]
I am not using borderless full screen, I am using regular full screen 
![Uploading image.jpg…]()

**Reply by Theighlin** [reply]
Non-exlusive fullscreen is not a setting, it means that when you're using fullscreen, it is not done so correctly.

**Reply by ghost** [reply]
I see, my misunderstanding then since I tried searching online
